style.css

```diff
@@ -347,3 +354,223 @@ footer {
   font-size: 22px;
   list-style-type: none;
 }
+
+/* ===== Desktop Version ===== */
+
+@media (min-width: 768px) {
+  /* ===== Navbar Section ===== */
+  .navBar {
+    padding: 0 142px 0 130px;
+    align-items: center;
+    height: 72px;
+  }
+
+  .navBtn {
+    display: flex;
+  }
+
+  .navBtn > .fa-bars {
+    display: none;
+  }
+
+  .navList {
+    display: flex;
+    color: #fff;
+    gap: 16px;
+    align-items: center;
+  }
+
+  .navList > li {
+    list-style-type: none;
+    font-size: 16px;
+    font-family: "Inter", sans-serif;
+  }
+
+  /* ===== Section One ===== */
+  .sectionOne {
+    background-image: url("images/Illustration-header.png");
+    background-repeat: no-repeat;
+    background-position: bottom right;
+    background-size: contain;
+    height: 100vh;
+  }
+
+  .aboutParent {
+    display: flex;
+    flex-direction: row-reverse;
+    gap: 65px;
+    padding: 100px 20px;
+    align-items: center;
+    justify-content: start;
+  }
```

Comment:
- [ ] Please try to make the images in this section align exactly as it does in the Figma design.

### Figma design
![Screenshot 2023-08-10 at 20 41 38 (2)](https://github.com/JimOluwaseyi/myPortfolio/assets/64774936/c34847c6-2021-4d9e-943c-cf587caafb50)


### Your design
<img width="1419" alt="Screenshot 2023-08-10 at 20 41 38" src="https://github.com/JimOluwaseyi/myPortfolio/assets/64774936/86f3b81b-4f5b-4f44-8d35-969cc91231a7">